### ISSUE TITLE
[JS][Rendering] ActionSet rendered DOM element shouldn't have the "menubar" role

### ISSUE THREAD

**Issue by dclaux** [microsoft/AdaptiveCards#6156]
# Target Platform

- NodeJS

# Problem Description

A card with an **ActionSet** that contains at least one **Action.OpenUrl** will not pass a fast-pass accessibility insight test because the DOM element generated for the ActionSet is stamped with the **"menubar"** role. A container with the "menubar" role can only contain children with roles “group”, “menuitemradio”, “menuitem” or “menuitemcheckbox”, however the DOM button generated for an Action.OpenUrl has the "link" role, which is correct.

The role of the DOM element generated for the ActionSet should be either removed or changed to something else (investigation needed). Note that removing the "menubar" role altogether solves the problem and makes the accessibility insight test pass; it may however not be the ideal solution in terms of end user experience.

# Version of SDK

Latest

**Comment by golddove** [comment]
@dclaux is this a dup of #6019 ?

**Comment by dclaux** [comment]
@golddove  Yep!
